src/borg/upgrade.py

```diff
@@ -129,7 +129,7 @@ def upgrade_zlib_and_level(meta, data):
             borg1_header_fmt = Struct(">I")
```

Comment:
i think this should be extracted so its only made once

Reply:
done